spec.html

```diff
@@ -11892,7 +11892,7 @@
           1. Let _tailCall_ be IsInTailPosition(_thisCall_).
           1. Return ? EvaluateDirectCall(_func_, _thisValue_, |Arguments|, _tailCall_).
         </emu-alg>
-        <p>A |CallExpression| evaluation that executes step 3.a.vi is a <dfn>direct eval</dfn>.</p>
+        <p>A |CallExpression| evaluation that executes step 3.a.vii is a <dfn>direct eval</dfn>.</p>
```

Comment:
Maybe you could have a special syntax for "step-specific notes". e.g.

    1. A step
      - ^NOTE: this is a note for the step

The positioning of the note (currently at the end of the algorithm) could be a view concern. This would make it easier to move steps around while avoiding the need for cross-referencing.